pkg/controller/che/che_controller.go

```diff
@@ -150,6 +151,49 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	var toRequestMapper handler.ToRequestsFunc = func(obj handler.MapObject) []reconcile.Request {
```

Comment:
Maybe we can move this watcher logic to the separated file?

Reply:
Done